model.py

```diff
@@ -4573,7 +4574,7 @@ def calculate_presentation_edition(self, policy=None):
 
     def calculate_presentation(
         self, policy=None, search_index_client=None, exclude_search=False,
-        default_fiction=False, default_audience=Classifier.AUDIENCE_ADULT
+        default_fiction=None, default_audience=None
```

Comment:
This _did_ matter -- the classifier itself didn't assume anything about fiction or audience status, but our actual entry point into that process did.